src/main/java/org/tron/common/storage/DepositImpl.java

```diff
@@ -603,13 +610,13 @@ private void commitContractCache(Deposit deposit) {
   }
 
   private void commitStorageCache(Deposit deposit) {
-    storageCache.forEach((key, value) -> {
+    storageCache.forEach((address, storage) -> {
       if (deposit != null) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6pnSgN4uCclo1b2&open=AWbJU6pnSgN4uCclo1b2)

[ISSUE_KEY]: AWbJU6pnSgN4uCclo1b2